### DISCUSSION TITLE
Surface Pro 1 Ivy Bridge working  with Brunch 126 and ChromeOS 126 but....

### DISCUSSION THREAD

**Discussion by rodpivo** [sebanc/brunch#2098] [General]
After following this https://github.com/sebanc/brunch/issues/2001 I managed to install on the Surface 1 Pro with samus-91 / brunch r91, everything worked, play, games, etc...

Then, I tested reef-119 with r119 (kernel 5.15) and the Play Store opens, install games , but games **do not load**, they open but close within a few seconds.  other apps work perfectly, YouTube Kids, Chrome, Gmail, cpu-z etc...

I upgraded to reef-126 and brunch r126 using the https://github.com/BiteDasher/brcr-update and with kernel 5.15 everything worked except games (just like it was with reef-119 and brunch r119) 
![WhatsApp Image 2024-07-05 at 11 26 07](https://github.com/sebanc/brunch/assets/174812315/dc7e3821-1b1e-40b8-97d6-4db07020767a)

I tried kernel 6.1, 6.5, 5.10, Chromebook 5.4, and even the Chromebook 4.19 kernel.

Kernels 6.x do not open the Play Store (even with android_init_fix and/or fix2) they stay with a black spin just like this https://github.com/sebanc/brunch/issues/813 and https://github.com/sebanc/brunch/issues/790, all the others kernels open and install apps normally, open web apps normally, but I still have the problem of games not loading...

If anyone has any idea how to fix the issue with games, this is the only thing missing!!!

![WhatsApp Image 2024-07-05 at 14 40 45](https://github.com/sebanc/brunch/assets/174812315/33c963d9-9b81-4ea0-bfaf-717142d02499)

![WhatsApp Image 2024-07-05 at 14 49 33](https://github.com/sebanc/brunch/assets/174812315/2d083c63-eeec-46d5-be17-afe1894f5dd3)

**Comment by maxed4901** [comment]
Not sure about surface pro 1 but i just manahed to install 126 larva on my sp3

**Reply by rodpivo** [reply]
> Not sure about surface pro 1 but i just manahed to install 126 larva on my sp3

Which base image (recovery) did you use for the Surface 3? I tried samus and reef on sp1, i didint find larva on https://cros.tech

**Reply by maxed4901** [reply]
What I meant was lava. The best success I had was with ekkoh 120 and brunch 120.. nissa and few others boot but have issues with playstore. I just installed nissa 126 on my i7 8th gen laptop. So far everything seems to be working.

**Reply by maxed4901** [reply]
Try asus chromebook c202SA
Hp Chromebook 11 G5

**Reply by rodpivo** [reply]
> Try asus chromebook c202SA Hp Chromebook 11 G5

i will try and post results here !!! tks !!

**Comment by madhits** [comment]
Did you look at all the coral board devices? See: https://cros.tech/board/coral/
You might get lucky with babytiger, https://cros.tech/device/babytiger/
I have a 4200u cpu which matches those systems but i have DDR3 not 4. My rammus r116 currently runs android 11 vs your 9, not sure if that is reported correctly because on r126 you should be a lot higher I would think. Perhaps thats an issue, idk. In any case there maybe enough instructions in your gen 3 intel to get you to where you are but just not enough emulation instructions that translate over to android.

**Reply by rodpivo** [reply]
> Did you look at all the coral board devices? See: https://cros.tech/board/coral/ You might get lucky with babytiger, https://cros.tech/device/babytiger/ I have a 4200u cpu which matches those systems but i have DDR3 not 4. My rammus r116 currently runs android 11 vs your 9, not sure if that is reported correctly because on r126 you should be a lot higher I would think. Perhaps thats an issue, idk. In any case there maybe enough instructions in your gen 3 intel to get you to where you are but just not enough emulation instructions that translate over to android.

i will try coral and babytiger and post results here !!! tks !!

**Reply by madhits** [reply]
Also ekko might work.
